content/censorship/snowflake-platforms/contents.lr

```diff
@@ -0,0 +1,10 @@
+_model: question
+---
+title: Which platforms is Snowflake supported on?
+---
+seo_slug: which-platforms-is-snowflake-supported-on
+---
+description:
+Snowflake is supported on Windows, MacOS and Linux.
```

Comment:
The seo_slug should be the directory name: snowflake-platforms
And fix here: 
Snowflake is supported on Tor Browser Alpha for macOS and GNU/Linux.
Windows and Android are still a work in progress.